samples/XCT.Sample.sln

```diff
@@ -1,6 +1,6 @@
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio Version 16
-VisualStudioVersion = 16.0.30204.135
+# Visual Studio Version 17
```

Comment:
Do we need to update this file?

Reply:
VS updated it automatically. Let's keep VS version 16.